datadog_checks_dev/tests/tooling/manifest_validator/input_constants.py

```diff
@@ -196,11 +198,208 @@
 
 DISPLAY_ON_PUBLIC_VALID_MANIFEST = JSONDict({"display_on_public_website": True})
 
-INVALID_DIFF = 'app_id id'
+IMMUTABLE_ATTRIBUTES_V1_MANIFEST = {"manifest_version": "1.0.0"}
+
+IMMUTABLE_ATTRIBUTES_V2_MANIFEST = JSONDict({"manifest_version": "2.0.0"})
+
+IMMUTABLE_ATTRIBUTES_PREV_MANIFEST_INVALID = json.dumps(
+    {
+        "app_id": "datadog-oracle",
+        "assets": {
+            "dashboards": {"oracle": "https://app.datadoghq.com/screen/integration/240/oracle-database---overview"},
```

Comment:
Is this intentionally a URL, or should this be `assets/dashboards/overview.json`?

Reply:
Fixed 👍🏼